### ISSUE TITLE
Masked fill error in NCE loss

### ISSUE THREAD

**Issue by Brechard** [taesungp/contrastive-unpaired-translation#34]
Hi!

First of all, amazing work, and thanks for making it available on Github.

I have found what seems a mistake when calculating Patch NCE loss, since in the comment above this line you state that the diagonal should be replaced with small values like exp(-10) but then in the mask fill it gets replace by -10, without the exponential.

https://github.com/taesungp/contrastive-unpaired-translation/blob/afdc8fb02742b4eef7724ca452fcee38016ee008/models/patchnce.py#L46

**Comment by taesungp** [comment]
The exponential is implicitly applied by the [CrossEntropyLoss ](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html), in [this line](https://github.com/taesungp/contrastive-unpaired-translation/blob/afdc8fb02742b4eef7724ca452fcee38016ee008/models/patchnce.py#L51).

**Comment by Brechard** [comment]
oh right! Thanks! 👍
